### DISCUSSION TITLE
Welcome to element Discussions!

### DISCUSSION THREAD

**Discussion by xinguanhua** [ElemeFE/element#22385] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by whidy** [comment]
I wish we can discussing with both English and Chinese here. English is best to comunicate, but I think most of developer here are Chinese, sometimes Chinese developers describe a problem with English maybe not accurate, and spend more time. My bit thinkings~

我希望大家能用双语沟通，如果是外国人，尽量使用英文，如果是中国人，根据能力来选择描述语言，有时候可能在某个问题上会因为开发者英文能力有限，造成描述不准确，反而增加理解难度，不好解决，当然能双语描述最佳了。哈哈哈我的一点想法。

**Reply by NidhoggDJoking** [reply]
维护者和使用者大多都是中国人的情况下使用中文可以避免一些不必要的问题，也没必要强调都用英语
比如想我英文能力有限，想要表达的东西一多就不得不借助翻译工具，然而翻译工具对一些专业词汇的翻译其实并不准确，平白增加了沟通成本。

**Comment by chz** [comment]
I wish Element to be responsive. Because it nominate for best component library for vue :)

**Comment by webvs2** [comment]
- [x] 添加icon文档的点击复制代码功能， #22426
- [ ]  💄💄💄  添加 **elementUi  Figma设计资源** 
- [ ] 迁移在线代码致其他平台

征求官方和社区的意见。
![image](https://user-images.githubusercontent.com/30883395/219543920-4e51a6b6-a8cf-4af8-ba18-951fc0d8e5e1.png)

**Reply by NidhoggDJoking** [reply]
> 我将添加icon的点击复制代码功能，征求官方和社区的意见。 ![image](https://user-images.githubusercontent.com/30883395/219365825-837f258f-d6ac-4203-a88d-53c9dc3df81f.png)

一个很棒的主意👍，尽管我一直使用iconfont提供的图标.建议`navigator.clipboard.writeText()`和`document.execCommand('copy')`2种方法都考虑进去因为`execCommand`不再推荐

**Comment by xiaotao2018** [comment]
**组件**：Table
**期望**：对于多列数据展示的表格，期望添加拖拽横向滑动查看数据的功能；益于多数据展示时不需要去寻找横向滚动条
**操作**：按下鼠标左键再向左或向右滑动来操作横向滚动条

**Component:** Table
**Expect:** For tables with multi-column data display, it is expected to add the function of dragging and sliding horizontally to view data. No need to find the horizontal scroll bar when displaying multiple data
**Operation:** Press the left mouse button and slide left or right to operate the horizontal scroll bar
